src/components/Input/Input.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import './Input.css'
+
+const Input = ({ inputClass, title, type, handleChange }) => (
+  <div className="input">
+    <label className={inputClass}>
+      <div className="input-title">{title}</div>
+
+      <div className="input-wrap">
+        <input type={type} onChange={e => handleChange(e)} />
+
+        <svg className="input-border" width="100%" height="100%"  xmlns="http://www.w3.org/2000/svg">
```

Comment:
this stands for an icon? why do you need SVG in input?

Reply:
Nope. Not an icon. It's a responsive svg for a visual effect. You can see the effect here when you focus the input:
https://codepen.io/eltonmesquita/pen/EbwrRE